src/components/deals-product-details/DealsProductDetails.tsx

```diff
@@ -10,7 +10,7 @@ import { getIndividualDealsProductDetails } from '../../services/DealsProductDet
 import { useDispatch } from 'react-redux'
 import { individualDealProductDetailData } from '../../redux/features/DealProductDetailSlice'
 import MoreDealsList from './MoreDealsList'
-import { storePageNumber } from '../../redux/features/dealModeSlice'
+import { shortByValue, storePageNumber } from '../../redux/features/dealModeSlice'
 
 const DealsProductDetails = () => {
```

Comment:
use `useCallback`

Reply:
sir, I made the change according to your comment but I have some doubts about whether I did it through right way or not.